kubetest/objects/serviceaccount.py

```diff
@@ -0,0 +1,90 @@
+"""Kubetest wrapper for the Kubernetes ``ServiceAccount`` API Object."""
+
+import logging
+
+from kubernetes import client
+
+from kubetest.objects import ApiObject
+
+log = logging.getLogger('kubetest')
+
+
+class ServiceAccount(ApiObject):
+    """Kubetest wrapper around a Kubernetes `ServiceAccount`_ API Object.
+    The actual ``kubernetes.client.V1ServiceAccount`` instance that this
+    wraps can be accessed via the ``obj`` instance member.
+    This wrapper provides some convenient functionality around the
+    API Object and provides some state management for the `ServiceAccount`_.
+    .. _ServiceAccount:
+        https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#serviceaccount-v1-core
+    """
+
+    obj_type = client.V1Namespace
```

Comment:
I believe this should be  `client.V1ServiceAccount`  instead

Reply:
Ahh the good ole copy pasta got me, let me get right on that!